static_src/test/functional/user_role.spec.js

```diff
@@ -1,59 +1,150 @@
 
-describe('User roles', function () {
-  let cookieResult;
-
-  beforeEach(function () {
-    browser.url('/');
-  });
+import UserRoleElement from './pageobjects/user_role.element';
 
-  afterEach(function () {
-    browser.deleteCookie('testing_user_role');
-  });
+describe('User roles', function () {
+  let cookieResult,
+    userRoleElement,
+    pageUrl,
+    cookieValue;
+  const guidManagerOrgX = 'org-manager-x-uid-601d-48c4-9705',
+    guidManagerOrgY = 'org-manager-y-uid-601d-48c4-9705',
+    cookieManagerOrgY = 'org_manager_org_y',
+    cookieManagerOrgX = 'org_manager_org_x',
+    cookieManagerSpaceXX = 'space_manager_space_xx',
+    // cookieManagerSpaceYY = 'space_manager_space_yy',
+    urlOrgY = '/#/org/user_role-org_y-ffe7-4aa8-8e85-94768d6bd250',
+    urlOrgX = '/#/org/user_role-org_x-ffe7-4aa8-8e85-94768d6bd250';
 
   describe('User role cookie test', function () {
+    afterEach(function () {
+      browser.deleteCookie('testing_user_role');
+    });
+
     describe('when cookie is set and deleted', function () {
       it('should reflect the cookie content', function () {
-        browser.setCookie({ name: 'testing_user_role', value: 'space_manager_space_xx' });
-        browser.url('/uaa/userinfo');
+        browser.url('/');
+        browser.setCookie({ name: 'testing_user_role', value: cookieManagerSpaceXX });
         cookieResult = browser.getCookie('testing_user_role').value;
-        expect(cookieResult).toBe('space_manager_space_xx');
+        expect(cookieResult).toBe(cookieManagerSpaceXX);
       });
     });
   });
 
-  describe('Org page', function () {
-    beforeEach(function () {
-      browser.url('/#/org/user_role-org_x-ffe7-4aa8-8e85-94768d6bd250');
+  describe('Org page as org manager X', function () {
+    describe('if org manager Y then they', function () {
+      it('should navigates to org Y ', function () {
+        browser.url(urlOrgY);
+
+        browser.waitForExist('.test-users');
+        userRoleElement = new UserRoleElement(browser, browser.element('.test-users'));
+
+        expect(userRoleElement.isVisible()).toBe(true);
+      });
+
+      it('should not be able to edit roles for org Y', function () {
+        expect(userRoleElement.isUserOrgManager(guidManagerOrgX)).toBe(false);
+        expect(userRoleElement.isUserOrgManager(guidManagerOrgY)).toBe(true);
+      });
     });
 
-    it('has a title', function () {
-      expect(browser.getTitle()).toBe('cloud.gov dashboard');
+    describe('if manager for org X then they', function () {
+      it('should navigates to org X ', function () {
+        browser.url(urlOrgX);
+
+        browser.waitForExist('.test-users');
+        userRoleElement = new UserRoleElement(browser, browser.element('.test-users'));
+
+        expect(userRoleElement.isVisible()).toBe(true);
+      });
+
+      it('should be able to edit roles for org X', function () {
+        expect(userRoleElement.isUserOrgManager(guidManagerOrgX)).toBe(true);
```

Comment:
i'm not sure if these assertions meet the test statement. maybe we should check for if the checkboxes are enabled / disabled